package.json

```diff
@@ -3,6 +3,7 @@
   "description": "Demo application for the pillarbox-web",
   "version": "1.0.0",
   "license": "MIT",
+  "type": "module",
```

Comment:
This or renaming the configuration file to `vite.config.mjs` is needed otherwise vite considers this a commonjs build. See https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated